src/transformers/models/vit/convert_vit_timm_to_pytorch.py

```diff
@@ -151,23 +151,37 @@ def convert_vit_checkpoint(vit_name, pytorch_dump_folder_path):
         config.patch_size = int(vit_name[-6:-4])
         config.image_size = int(vit_name[-3:])
     # size of the architecture
-    if vit_name[4:].startswith("small"):
-        config.hidden_size = 768
-        config.intermediate_size = 2304
-        config.num_hidden_layers = 8
-        config.num_attention_heads = 8
-    if vit_name[4:].startswith("base"):
-        pass
-    elif vit_name[4:].startswith("large"):
-        config.hidden_size = 1024
-        config.intermediate_size = 4096
-        config.num_hidden_layers = 24
-        config.num_attention_heads = 16
-    elif vit_name[4:].startswith("huge"):
-        config.hidden_size = 1280
-        config.intermediate_size = 5120
-        config.num_hidden_layers = 32
-        config.num_attention_heads = 16
+    if "deit" in vit_name:
```

Comment:
Should DeiT be handled by the ViT conversion script? I would put that in the DeiT conversion script aboee

Reply:
The story is a bit complicated: deit has checkpoints which are just more efficient ViT models, hence I converted those using the original ViT conversion script (and added "deit" there), and there are also distilled checkpoints, which are converted in `convert_deit_timm_to_pytorch`.py.

Reply:
Hmmm okay, as long as that's very well documented that works for me